dev-tools/multipass-cloud-init.yml

```diff
@@ -0,0 +1,48 @@
+#cloud-init
```

Comment:
For my information, why where, multipass is used here?

Reply:
I was trying to use it to ensure test ran on linux as I had previous issues with the vagrantbox